README.md

```diff
@@ -94,22 +96,24 @@ Choose from a variety of supported database drivers:
 - [Redis](https://github.com/redis/go-redis)
 
 <a id="advanced-features"></a>
+
 <h2>
   <picture>
     <img src="./public/advanced.gif?raw=true" width="70px" style="margin-right: 1px;">
   </picture>
   Advanced Features
 </h2>
 
-Blueprint is focused on being as minimalistic as possible. That being said, we wanted to offer the ability to add other features people may want without bloating the overall experience. 
+Blueprint is focused on being as minimalistic as possible. That being said, we wanted to offer the ability to add other features people may want without bloating the overall experience.
 
 You can now use the `--advanced` flag when running the `create` command to get access to the following features. This is a multi-option prompt; one or more features can be used at the same time:
 
 - [HTMX](https://htmx.org/) support using [Templ](https://templ.guide/)
 - CI/CD workflow setup using [Github Actions](https://docs.github.com/en/actions)
-
+- [Websocket](https://en.wikipedia.org/wiki/WebSocket) sets up a websocket endpoint
```

Comment:
I think this needs to either link to some Go related source material and not just a wikipedia article on web sockets

Reply:
I could link the most used websocket library [^1] in the generated code but that would not encompass fiber as an option as that is using it's own websocket implementation.

Is that okay with you?

[^1]: https://pkg.go.dev/nhooyr.io/websocket